lib/entropy/db/sqlite.py

```diff
@@ -700,6 +700,7 @@ def vacuum(self):
         """
         Reimplemented from EntropySQLRepository.
         """
+        self._connection().commit()
```

Comment:
self.commit()

Reply:
Please mention the reason why we issue self.commit() before vacuum() in a comment.